bliss/simulator/decoder.py

```diff
@@ -1,153 +1,25 @@
-from pathlib import Path
 from typing import Tuple
 
 import galsim
 import numpy as np
 import torch
-from astropy.io import fits
-from einops import rearrange, reduce
-from omegaconf import DictConfig
 from torch import nn
 
 from bliss.catalog import SourceType, TileCatalog
-from bliss.surveys.sdss import SDSSDownloader
 
 
 class ImageDecoder(nn.Module):
     def __init__(
         self,
-        pixel_scale: float,
-        psf_slen: int,
-        sdss_fields: DictConfig,
+        psf,
+        bands: Tuple[int, ...] = (0, 1, 2, 3, 4),
```

Comment:
Same here, we should just always require passing bands instead of having a default value

Reply:
Done.